app/src/main/java/org/oppia/app/player/exploration/ExplorationActivity.kt

```diff
@@ -130,12 +145,20 @@ class ExplorationActivity : InjectableAppCompatActivity(), StopExplorationInterf
     allHintsExhausted: Boolean
   ) {
     if (getHintsAndSolution() == null) {
-      val hintsAndSolutionFragment = HintsAndSolutionFragment.newInstance(explorationId, newAvailableHintIndex, allHintsExhausted)
+      val hintsAndSolutionFragment = HintsAndSolutionFragment.newInstance(
+        explorationId,
+        newAvailableHintIndex,
+        allHintsExhausted
+      )
       hintsAndSolutionFragment.showNow(supportFragmentManager, TAG_HINTS_AND_SOLUTION_DIALOG)
     }
   }
 
   override fun dismiss() {
     getHintsAndSolution()?.dismiss()
   }
+
+  override fun onDeafultFontSizeLoaded(result: StoryTextSize) {
```

Comment:
Instead of `result` we can name it something more contextual `storyTextSize` or `defaultStoryTextSize`

Reply:
updated to storyTextSize.

Reply:
You will need to make this change in `DeafultFontSizeStateListener` interface also.

Reply:
Done.